### ISSUE TITLE
question about feature X&H in a cell

### ISSUE THREAD

**Issue by dmy1997** [yanwei-li/DynamicRouting#4]
Hello, i don't understand the necessity of eq.1 in your paper.
Why not use X to upsample/keep/downsample directly? Is it hard to train?

**Comment by yanwei-li** [comment]
Operations (SeqConv 3x3) in Equ. 1 are adopted for aggregating input features. If all of the operations in Equ. 1 are dropped, there will be no parameters in the routing space, which may result in the network collapse. Actually, if all the paths in one node are closed (sum(\alpha_s_j)=0 in Equ. 5), the H_s will equal to X_s, and the operations in this node will be dropped to save resource.

**Comment by dmy1997** [comment]
I see. Thx.

**Comment by dmy1997** [comment]
Hello, do you have the problem that it's hard to train. Maybe because it's too dynamic? We need to decrease learning rate or increase iteration num compared to usual cases.

**Comment by yanwei-li** [comment]
Hi! Training from scratch with a short period may be unstable in some cases. ImageNet pretrained model is recommended for training.

**Comment by dmy1997** [comment]
Hi! I have another question. Why you set the bias of Cell weights 1.5 ( ''gate_bias=1.5'')? How to decide the value?

**Comment by yanwei-li** [comment]
We'd like to initialize gates with activated prob > 0.9. So, we initialize the bias with 1.5 (`tanh(1.5)=0.905`), which is proved to have better performance in the experiments.
